test/e2e/page-objects/flows/send-transaction.flow.ts

```diff
@@ -36,7 +36,43 @@ export const sendTransaction = async (
 
   // confirm transaction when user lands on confirm transaction screen
   const confirmTxPage = new ConfirmTxPage(driver);
-  await confirmTxPage.check_pageIsLoaded(gasfee, totalfee);
+  await confirmTxPage.check_pageIsLoaded(gasFee, totalFee);
+  await confirmTxPage.confirmTx();
+};
+
+/**
+ * This function initiates the steps required to send a transaction from the homepage to final confirmation.
+ *
+ * @param driver - The webdriver instance.
+ * @param recipientAccount - The recipient account.
+ * @param amount - The amount of the asset to be sent in the transaction.
+ * @param gasFee - The expected transaction gas fee.
+ * @param totalFee - The expected total transaction fee.
+ */
+export const sendTransactionToAccount = async (
+  driver: Driver,
+  recipientAccount: string,
+  amount: string,
+  gasFee: string,
+  totalFee: string,
```

Comment:
the same comment as above, about using an object as params. In this case, this is also only used in 2 specs now, so if we want to do this change, I think it is a good moment now

Reply:
Thanks for the suggestion, addressed in [0aa97f0](https://github.com/MetaMask/metamask-extension/pull/28325/commits/0aa97f0cfa2bba99b73f4933263a3ba03c915340)